src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -272,6 +272,124 @@ const listifyCenters = (center, idx, list) => {
   );
 };
 
+export const documentTypeEnums = [
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  '10',
+  '11',
+  '12',
+  '13',
+  '14',
+  '15',
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+  '21',
+  '22',
+  '23',
+  '24',
+  '25',
+  '26',
+  '27',
+  '28',
+  '29',
+  '30',
+  '31',
+  '32',
+  '33',
+  '34',
+  '35',
+  '36',
+  '37'
+];
+
+export const documentTypes = [
+  'Application for Automotive Adaptive Equipment',
+  'Application for Compensation for Unemployability',
+  'Application for Specially Adapted Housing',
+  'Examination for Housebound Status',
+  'Request for Nursing Home Information',
+  'Examination for Housebound Status',
+  'Temporary Total Disability for PTSD_21_0781 and PTSD_21_0781A',
+  'Service Connected PTSD Statement',
+  'Service Connected Personal Assault PTSD Statement',
+  'Police Report',
+  'Military Performance Report',
+  'ER Medical Report',
+  'Private Medical Treatment Record',
+  'Buddy Statement',
+  'DD214',
+  'Military Personnel File',
+  'Decorations/Awards/Medal Citations',
+  'Other Correspondence', // corrects typo in original list
+  'Personal Journal or Diary',
+  'Certificate of Release or Discharge from Active Duty (e.g. DD 214, NOAA 56-16, PHS 1867)',
+  'Service Treatment Record - Dental',
+  'Service Treatment Record - Medical',
+  'Medical Evidence from Private Provider',
+  'Medical Evidence from Government Facility',
+  'Social Security Disability Documentation',
+  'Military Personnel Record',
+  'Buddy/Lay Statement',
+  'VA Form 21-4142 - Authorization To Disclose Information',
+  'VA Form 21-4142a - General Release for Medical Provider Information',
+  'Other Correspondence',
+  'Application to Claim Dependents',
+  'Disability Benefits Questionnaire (DBQ) - Veteran Provided',
+  'Disability Benefits Questionnaire (DBQ) - Veteran Provided',
+  'Photographs',
+  'Disability Benefits Questionnaire (DBQ) - Veteran Provided',
+  'Goldmann Perimetry Chart/Field Of Vision Chart'
```

Comment:
@egarreau it looks like this list has more items than https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/models/evss_claim_document.rb#L23-L45, should some of these be removed?

Reply:
no, the back-end list needs to be expanded to include everything from this list https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Disability%20Compensation/engineering/EVSS/526DocTypes.xlsx